### PR TITLE
Remove link previews

### PR DESCRIPTION
**Pull request by megrogan**
Latest is on webtest

### CODE REVIEW

File: frontend/openchat-client/src/utils/linkPreviews.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
+const LINKS_REGEX = new RegExp("<link[^>]*>", "g");
+const URL_REGEX = new RegExp(`(https?://[^\\s)]+)`, "g");
+
+export function insertLinkTags(text: string): string {
+    return text.replace(URL_REGEX, (match) => buildLinkTag(match, true));
+}
+
+export function stripLinkTags(text: string): string {
+    if (text.includes("<link ")) {
```
[the remark this answers is not in the document]
Comment (megrogan) [reply frontend/openchat-client/src/utils/linkPreviews.ts:9]: Because it is more efficient than regex

[the remark this answers is not in the document]
Comment (julianjelfs) [reply frontend/openchat-client/src/utils/linkPreviews.ts:9]: I'm very skeptical that that is consideration that trumps the unpleasantness of having the thing that you are looking for defined in two (or _three_) places.

File: frontend/app/src/components/home/LinkPreviews.svelte [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        class:visible={preview.kind !== "generic"}
+        class:me>
+        {#if me}
+            <div class="remove" class:rtl={$rtlStore} on:click={() => removePreview(preview)}>
+                <CloseIcon viewBox="0 0 24 24" size={$iconSize} color={"var(--button-txt)"} />
+            </div>
+        {/if}
+        <div class="inner">
+            {#if preview.kind === "twitter"}
+                <Tweet tweetId={preview.tweetId} {intersecting} />
```
Comment (julianjelfs) [inline frontend/app/src/components/home/LinkPreviews.svelte:145]: So we should still have `on:rendered` against the `Tweet` component now shouldn't we?

Comment (megrogan) [reply frontend/app/src/components/home/LinkPreviews.svelte:145]: Oops - put it on YouTubePreview by mistake!

**Review by julianjelfs — APPROVED** [review]

**Pull request merged**